eppo/src/main/java/cloud/eppo/android/ConfigurationRequestor.java

```diff
@@ -24,45 +24,50 @@ public ConfigurationRequestor(ConfigurationStore configurationStore, EppoHttpCli
     }
 
     public void load(InitializationCallback callback) {
-        AtomicBoolean cachedUsed = new AtomicBoolean(false);
+        AtomicBoolean callbackCalled = new AtomicBoolean(false);
```

Comment:
Instead of tracking cache usage, we just track if the callback has been called yet. This way both fetch won't call the callback if the cache load finished first, and the cache load won't call the callback if the fetch finished first.